lib_pypy/_sqlite3.py

```
@@ -379,7 +467,13 @@ class Connection(object):
     _db = None
 
     def __init__(self, database, timeout=5.0, detect_types=0, isolation_level="",
-                 check_same_thread=True, factory=None, cached_statements=100, uri=0):
+                 check_same_thread=True, factory=None, cached_statements=100,
+                 uri=0, *, autocommit=LEGACY_TRANSACTION_CONTROL):
+        if (autocommit is not True and autocommit is not False and
+                autocommit != LEGACY_TRANSACTION_CONTROL):
+            raise ValueError(
+                "autocommit must be True, False, or "
+                "sqlite3.LEGACY_TRANSACTION_CONTROL")
```

Ruff (0.16.2)

[warning] Avoid specifying long messages outside the exception class

(TRY003)